packages/app-project/.babelrc

```diff
@@ -2,12 +2,24 @@
   "plugins": [
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
     "@babel/plugin-proposal-optional-chaining",
-    "babel-plugin-styled-components",
     "dynamic-import-node"
   ],
   "presets": ["next/babel"],
   "env": {
+    "dev": {
```

Comment:
People were saying this was necessary for the styled-components babel plugin in this issue: https://github.com/zeit/next.js/issues/3706

Basically, their plugin is buggy and supposedly this will help resolve some of the collisions? I'm not sure since there's still an issue with the footer, but the header appears to be working in this branch now.